kubernetes-client/src/main/java/io/fabric8/kubernetes/client/internal/CertUtils.java

```diff
@@ -284,10 +277,21 @@ public static KeyStore createKeyStore(String clientCertData, String clientCertFi
 
     while ((line = reader.readLine()) != null) {
       if (line.indexOf(endMarker) != -1) {
-        return ByteString.decodeBase64(buf.toString()).toByteArray();
+        return Base64.getDecoder().decode(buf.toString());
       }
       buf.append(line.trim());
     }
     throw new IOException("PEM is invalid : No end marker");
   }
+
+  private static ByteArrayInputStream createInputStreamFromBase64DecodedString(String data) {
```

Comment:
nit: shouldn't this be named `createInputStreamFromBase64EncodedString` or `createDecodedInputStreamFromBase64EncodedString` ?

Reply:
Maybe just `createInputStream` and specify in the javadoc that this method returns a decoded inputstream for the provided optionally base64 encoded String